### DISCUSSION TITLE
v1.27.4-rc1+rke2r1 missing files in release, even though they are there in the tag

### DISCUSSION THREAD

**Discussion by taliesins** [rancher/rke2#4520] [Q&A]
@briandowns I think something strange has broken in the release.

I was hoping to download the latest release of rke2 (v1.27.4-rc1+rke2r1) and I received:
```
[INFO]  downloading checksums at https://github.com/rancher/rke2/releases/download/v1.27.4+rke2r1/sha256sum-amd64.txt
curl: (22) The requested URL returned error: 404
```
Looking at the releases page it seems files are missing from the release:
![image](https://github.com/rancher/rke2/assets/314567/cd0ad51c-759b-40f6-bc4c-e5842233d611)

**Comment by brandond** [comment]
The release isn't done yet, due to CI issues. That's why it's still tagged pre-release. When CI is complete, the pre-release tag will be removed.

**Reply by brandond** [reply]
If you wanted to install `v1.27.4-rc1+rke2r1` that's not what you're actually doing; it appears you are trying to install `v1.27.4+rke2r1` which is still in the process of being released. `v1.27.4-rc1+rke2r1` should have all the correct assets available, if you want to continue to use that while the final release builds.
